src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/ManageBookmarks/ManageBookmarksVB.xaml.vb

```diff
@@ -0,0 +1,166 @@
+'
+' Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an 
+' "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific 
+' language governing permissions and limitations under the License.
+
+Imports Esri.ArcGISRuntime.Mapping
+Imports System.Windows.Controls
+
+Namespace ManageBookmarks
+
+    Partial Public Class ManageBookmarksVB
+
+        Public Sub New()
+            InitializeComponent()
+
+            ' Create the UI, setup the control references and execute initialization. 
+            Initialize()
+
+        End Sub
+
+        Private Sub Initialize()
+
+            ' Create new map with a basemap.
+            Dim myMap As New Map(Basemap.CreateImageryWithLabels())
+
+            ' Set the mapview, map property to the basemap.
+            MyMapView.Map = myMap
+
+            ' Create a set of predefined bookmarks; each one follows the pattern of:
+            ' ~ Initialize a viewpoint pointing to a latitude longitude
+            ' ~ Create a new bookmark
+            ' ~ Give the bookmark a name
+            ' ~ Assign the viewpoint
+            ' ~ Add the bookmark to bookmark collection of the map
+            ' ~ Add the bookmark name to the UI combobox for the user to choose from 
+
+            ' Bookmark-1
+            Dim myViewpoint1 As New Viewpoint(27.3805833, 33.6321389, 6000)
+            Dim myBookmark1 As New Bookmark()
+            myBookmark1.Name = "Mysterious Desert Pattern"
+            myBookmark1.Viewpoint = myViewpoint1
+            MyMapView.Map.Bookmarks.Add(myBookmark1)
+            bookmarkChooser.Items.Add(myBookmark1.Name)
+
+            ' Bookmark-2
+            Dim myViewpoint2 As New Viewpoint(37.401573, -116.867808, 6000)
+            Dim myBookmark2 As New Bookmark()
+            myBookmark2.Name = "Strange Symbol"
+            myBookmark2.Viewpoint = myViewpoint2
+            MyMapView.Map.Bookmarks.Add(myBookmark2)
+            bookmarkChooser.Items.Add(myBookmark2.Name)
+
+            ' Bookmark-3
+            Dim myViewpoint3 As New Viewpoint(-33.867886, -63.985, 40000)
+            Dim myBookmark3 As New Bookmark()
+            myBookmark3.Name = "Guitar-Shaped Forest"
+            myBookmark3.Viewpoint = myViewpoint3
+            MyMapView.Map.Bookmarks.Add(myBookmark3)
+            bookmarkChooser.Items.Add(myBookmark3.Name)
+
+            ' Bookmark-4
+            Dim myViewpoint4 As New Viewpoint(44.525049, -110.83819, 6000)
+            Dim myBookmark4 As New Bookmark()
+            myBookmark4.Name = "Grand Prismatic Spring"
+            myBookmark4.Viewpoint = myViewpoint4
+            MyMapView.Map.Bookmarks.Add(myBookmark4)
+            bookmarkChooser.Items.Add(myBookmark4.Name)
+
+            ' Set the initial combobox selection to the lat bookmark added.
+            bookmarkChooser.SelectedIndex = 3
```

Comment:
Suggestion: set the selected item with logic to find the last bookmark (rather than hard-coding it).
````
bookmarkChooser.SelectedItem = MyMapView.Map.Bookmarks.Last()
````